app/jsx/components/SubheaderComp.jsx

```diff
@@ -48,50 +52,50 @@ class SubheaderComp extends React.Component {
 
     // For series and journal pages, a basic one-screened wizard will be used when directSubmit = disabled/moribund. 
     // Otherwise they link direct to subi (or directSubmit URL if that's defined)
-    if (p.unit.type == 'journal' || p.unit.type.includes('series')) {
-      let buttonName = (p.unit.type == 'journal') ? "Submit" : "Deposit"
-      if (["moribund", "disabled", "hide"].includes(p.directSubmit)) {
+    if (unit.type == 'journal' || unit.type.includes('series')) {
+      let buttonName = (unit.type == 'journal') ? "Submit" : "Deposit"
+      if (["moribund", "disabled", "hide"].includes(h.directSubmit)) {
         depositButton = <button id="wizardlyDeposit" className="o-button__3" onClick={(event)=>{
                                  this.setState({modalOpen:true})
                                  event.preventDefault()} } >{buttonName}</button>
         wizard = (<WizardInertComp showModal={this.state.modalOpen}
                                    parentSelector={()=>$('#wizardModalBase')[0]}
-                                   onCancel={e=>this.closeWizardModal(e)} campusName={p.campusName}
-                                   type={p.unit.type} directSubmit={p.directSubmit} />)
+                                   onCancel={e=>this.closeWizardModal(e)} campusName={h.campusName}
+                                   type={unit.type} directSubmit={h.directSubmit} />)
 
       } else {
-        depositButton = (<a className="o-download__button" href={p.directSubmitURL ? p.directSubmitURL : "https://submit.escholarship.org/subi/directSubmit?target="+p.unit.id}>{buttonName}</a>)
+        depositButton = (<a className="o-download__button" href={h.directSubmitURL ? h.directSubmitURL : "https://submit.escholarship.org/subi/directSubmit?target="+unit.id}>{buttonName}</a>)
```

Comment:
I'll be filling in these directSubmit URLs shortly. Cool that the support's already in for them!